app/main/views/services.py

```diff
@@ -408,10 +408,79 @@ def add_service_type_price():
     price = ServiceTypePrice()
     price.update_from_json(price_json['price'])
 
+    today = Pendulum.today()
+    tomorrow = Pendulum.tomorrow()
+    service_type_prices = ServiceTypePrice.query.filter(
+        ServiceTypePrice.supplier_code == price.supplier_code,
+        ServiceTypePrice.service_type_id == price.service_type_id,
+        ServiceTypePrice.sub_service_id == price.sub_service_id,
+        ServiceTypePrice.region_id == price.region_id
+    ).all()
+
+    # look for current price
+    current_service_type_price = None
+    current_service_type_prices = [c for c in service_type_prices if c.date_from < today and c.date_to >= today]
+    current_service_type_prices_length = len(current_service_type_prices)
+    if current_service_type_prices_length > 1:
+        return jsonify(CurrentServiceTypePrices=current_service_type_prices,
+                       msg="There is more than one current price",
+                       error=True), 400
```

Comment:
should the http verb indicate an error?

Reply:
i believe so, because this indicates a more serious problem and should not be buried with the 2xx http status codes